noobaa_sa/bucket.py

```diff
@@ -0,0 +1,121 @@
+"""
+Module which contain bucket operations like create, delete, list and update
+"""
+
+import logging
+import re
+
+from framework import config
+from framework.connection import SSHConnection
+from noobaa_sa.defaults import MANAGE_NSFS
+import noobaa_sa.exceptions as e
+
+log = logging.getLogger(__name__)
+
+
+
+
+class BucketOperation:
+    """
+    Bucket operations
+    """
+    def __init__(self):
+        """
+        Initialize necessary variable
+        """
+        self.manage_nsfs = MANAGE_NSFS
+        self.config_root = config.ENV_DATA["config_root"]
+        self.conn = SSHConnection().connection
+
+    def createBucket(self, account_name, bucket_name, config_root=None):
+        """
+        Create bucket using CLI
+
+        Args:
+            config_root (str): Path to config root
+            account_name: User name
+            bucket_name: Name of the bucket
+        """
+        unwanted_log = "2>/dev/null"
+        base_cmd = f"sudo /usr/local/noobaa-core/bin/node {self.manage_nsfs} "
+        if config_root is None:
+            config_root = self.config_root
+        log.info("Gather user info before creating bucket")
+        cmd = f"{base_cmd}account status --config_root {config_root} --name {account_name} {unwanted_log}"
```

Comment:
Please adjust the spaces here to better stick with the existing convention:
```suggestion
        base_cmd = f"sudo /usr/local/noobaa-core/bin/node {self.manage_nsfs}"
        if config_root is None:
            config_root = self.config_root
        log.info("Gather user info before creating bucket")
        cmd = f"{base_cmd} account status --config_root {config_root} --name {account_name} {unwanted_log}"
```

Reply:
Fixed